integration/test/app/basic/basic_test.go

```diff
@@ -39,10 +39,22 @@ func TestAppLifecycle(t *testing.T) {
 	var err error
 
 	{
-		config.Logger.Debugf(ctx, "installing %#q", key.ChartOperatorUniqueName())
+		{
+			crdName := "Chart"
+			config.Logger.Debugf(ctx, "ensuring %#q CRD exists", crdName)
+
+			err := config.K8sClients.CRDClient().EnsureCreated(ctx, crd.LoadV1("application.giantswarm.io", crdName), backoff.NewMaxRetries(7, 1*time.Second))
+			if err != nil {
+				t.Fatalf("expected %#v got %#v", nil, err)
+			}
+
+			config.Logger.Debugf(ctx, "ensured %#q CRD exists", crdName)
```

Comment:
Creating the chart CRD needs to happen first or the readiness probe in chart-operator will fail.